test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/RestoreCommandTests.cs

```diff
@@ -1212,6 +1212,90 @@ public async Task Restore_WithHttpSource_Warns()
             Assert.Contains("You are running the 'restore' operation with an 'http' source, 'http://api.source/index.json'. Support for 'http' sources will be removed in a future version.", result.Output);
         }
 
+        [Theory]
+        [InlineData("false")]
+        [InlineData("FALSE")]
+        [InlineData("invalidString")]
+        [InlineData("")]
+        public async Task Restore_WithHttpSourceAndFalseAllowInsecureConnections_WarnsCorrectly(string allowInsecureConnections)
+        {
+            // Arrange
+            using var pathContext = new SimpleTestPathContext();
+            // Set up solution, project, and packages
+            var solution = new SimpleTestSolutionContext(pathContext.SolutionRoot);
+            var packageA = new SimpleTestPackageContext("a", "1.0.0");
+            await SimpleTestPackageUtility.CreateFolderFeedV3Async(pathContext.PackageSource, packageA);
+            pathContext.Settings.AddSource("http-feed", "http://api.source/index.json", allowInsecureConnections);
+            pathContext.Settings.AddSource("https-feed", "https://api.source/index.json", allowInsecureConnections);
+
+            var net461 = NuGetFramework.Parse("net461");
+            var projectA = new SimpleTestProjectContext(
+                "a",
+                ProjectStyle.PackagesConfig,
+                pathContext.SolutionRoot);
+            projectA.Frameworks.Add(new SimpleTestProjectFrameworkContext(net461));
+            var projectAPackages = Path.Combine(pathContext.SolutionRoot, "packages");
+
+            Util.CreateFile(Path.GetDirectoryName(projectA.ProjectPath), "packages.config",
+@"<packages>
+  <package id=""A"" version=""1.0.0"" targetFramework=""net461"" />
+</packages>");
+
+            solution.Projects.Add(projectA);
+            solution.Create(pathContext.SolutionRoot);
+
+            // Act
+            var result = RunRestore(pathContext, _successExitCode);
+
+            // Assert
+            result.Success.Should().BeTrue();
+            Assert.Contains($"Added package 'A.1.0.0' to folder '{projectAPackages}'", result.Output);
+            Assert.Contains("You are running the 'restore' operation with an 'http' source, 'http://api.source/index.json'. Support for 'http' sources will be removed in a future version.", result.Output);
```

Comment:
The string in `Warning_HttpServerUsage` uses uppercase `HTTP`. I would think this test would fail, unless Contains is case insensitive.

Reply:
Thanks for reviewing @donnie-msft ! Good catch!
There are 5 `Warning_HttpServerUsage` in our codebase: https://github.com/search?q=repo%3ANuGet%2FNuGet.Client%20name%3D%22Warning_HttpServerUsage%22&type=code
And only one of the 5 is: 
https://github.com/NuGet/NuGet.Client/blob/6e066f0535191e528a724db05758e29982fc909a/src/NuGet.Core/NuGet.PackageManagement/Strings.resx#L433
All other 4 are the same.
The only different one is brought in in PR https://github.com/NuGet/NuGet.Client/pull/4583/files
May I know if there is any reason to have this one different?
Or shall I change it to be the same with the rest 4?

Reply:
I suggest using uppercase in all of them, since it's an acronym. I'm not sure if it's considered a breaking change to edit a warning message? If so, maybe not worth doing at this point?

Reply:
Thanks @donnie-msft ! Checked with the team and it's not a breaking change. Fixed.